.gitignore

```diff
@@ -159,3 +159,4 @@ src/wazuh_modules/vulnerability_detector/msu.json.gz
 # Flatbuffers generated headers.
 src/shared_modules/utils/flatbuffers/include/
 src/wazuh_modules/syscollector/include/
+src/wazuh_modules/vulnerability_scanner/include/*_generated.h
```

Comment:
I think regex is not supported fully in `.gitignore`, probably have to add each file to skip them from the VCS.

Reply:
It does work, it also used some lines above. This is the outcome 
![image](https://github.com/wazuh/wazuh/assets/13010397/212cad2e-ce80-44ac-b813-1ea0f9b9fc07)